embedchain/vectordb/lancedb.py

```diff
@@ -0,0 +1,281 @@
+from typing import Any, Dict, List, Optional, Union
+
+import pyarrow as pa
+
+try:
+    import lancedb
+except ImportError:
+    raise ImportError("LanceDB is required. Install with `pip install lancedb`") from None
```

Comment:
Here you can do `pip install "embedchain[lancedb]"`

Reply:
done